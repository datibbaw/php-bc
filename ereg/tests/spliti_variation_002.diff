006+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
006- Error: 8192 - Function spliti() is deprecated, %s(74)
013+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
013- Error: 8192 - Function spliti() is deprecated, %s(74)
020+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
020- Error: 8192 - Function spliti() is deprecated, %s(74)
027+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
027- Error: 8192 - Function spliti() is deprecated, %s(74)
034+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
034- Error: 8192 - Function spliti() is deprecated, %s(74)
041+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
041- Error: 8192 - Function spliti() is deprecated, %s(74)
048+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
048- Error: 8192 - Function spliti() is deprecated, %s(74)
055+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
055- Error: 8192 - Function spliti() is deprecated, %s(74)
064+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
064- Error: 8192 - Function spliti() is deprecated, %s(74)
072+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
073+ Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
074+ bool(false)
075+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)
076+ 
077+ Arg value Array 
078+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
079+ Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
080+ bool(false)
081+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)
082+ 
083+ Arg value Array 
084+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
085+ Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
086+ bool(false)
087+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)
088+ 
089+ Arg value Array 
090+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
091+ Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
092+ bool(false)
072- Error: 8192 - Function spliti() is deprecated, %s(74)
073- Error: 2 - spliti() expects parameter 2 to be string, array given, %s(74)
074- NULL
078- Error: 8192 - Function spliti() is deprecated, %s(74)
079- Error: 2 - spliti() expects parameter 2 to be string, array given, %s(74)
080- NULL
081- Error: 8 - Array to string conversion, %sspliti_variation_002.php(%d)
083- Arg value Array 
084- Error: 8192 - Function spliti() is deprecated, %s(74)
085- Error: 2 - spliti() expects parameter 2 to be string, array given, %s(74)
086- NULL
087- Error: 8 - Array to string conversion, %sspliti_variation_002.php(%d)
088- 
089- Arg value Array 
096+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
097+ Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
098+ bool(false)
099+ 
100+ Arg value  
101+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
102+ array(1) {
103+   [0]=>
104+   string(0) ""
105+ }
106+ 
107+ Arg value  
108+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
109+ array(1) {
110+   [0]=>
111+   string(0) ""
112+ }
113+ 
114+ Arg value 1 
115+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
116+ array(1) {
117+   [0]=>
118+   string(1) "1"
119+ }
120+ 
121+ Arg value  
122+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
123+ array(1) {
124+   [0]=>
125+   string(0) ""
126+ }
127+ 
128+ Arg value 1 
129+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
130+ array(1) {
131+   [0]=>
132+   string(1) "1"
133+ }
134+ 
135+ Arg value  
136+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
137+ array(1) {
138+   [0]=>
139+   string(0) ""
140+ }
141+ 
142+ Arg value  
143+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
144+ array(1) {
145+   [0]=>
146+   string(0) ""
147+ }
148+ 
149+ Arg value  
150+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
151+ array(1) {
152+   [0]=>
153+   string(0) ""
154+ }
155+ Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)
156+ 
157+ Arg value  
158+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
159+ Error: 2 - preg_split() expects parameter 2 to be string, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
160+ bool(false)
161+ 
162+ Arg value  
163+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
164+ array(1) {
165+   [0]=>
166+   string(0) ""
167+ }
169+ Arg value  
170+ Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
171+ array(1) {
172+   [0]=>
173+   string(0) ""
174+ }
175+ Done
090- Error: 8192 - Function spliti() is deprecated, %s(74)
091- Error: 2 - spliti() expects parameter 2 to be string, array given, %s(74)
092- NULL
093- Error: 8 - Array to string conversion, %sspliti_variation_002.php(%d)
094- 
095- Arg value Array 
096- Error: 8192 - Function spliti() is deprecated, %s(74)
097- Error: 2 - spliti() expects parameter 2 to be string, array given, %s(74)
098- NULL
099- 
100- Arg value  
101- Error: 8192 - Function spliti() is deprecated, %s(74)
102- array(1) {
103-   [0]=>
104-   string(0) ""
105- }
106- 
107- Arg value  
108- Error: 8192 - Function spliti() is deprecated, %s(74)
109- array(1) {
110-   [0]=>
111-   string(0) ""
112- }
113- 
114- Arg value 1 
115- Error: 8192 - Function spliti() is deprecated, %s(74)
116- array(1) {
117-   [0]=>
118-   string(1) "1"
119- }
120- 
121- Arg value  
122- Error: 8192 - Function spliti() is deprecated, %s(74)
123- array(1) {
124-   [0]=>
125-   string(0) ""
126- }
127- 
128- Arg value 1 
129- Error: 8192 - Function spliti() is deprecated, %s(74)
130- array(1) {
131-   [0]=>
132-   string(1) "1"
133- }
134- 
135- Arg value  
136- Error: 8192 - Function spliti() is deprecated, %s(74)
137- array(1) {
138-   [0]=>
139-   string(0) ""
140- }
141- 
142- Arg value  
143- Error: 8192 - Function spliti() is deprecated, %s(74)
144- array(1) {
145-   [0]=>
146-   string(0) ""
147- }
148- 
149- Arg value  
150- Error: 8192 - Function spliti() is deprecated, %s(74)
151- array(1) {
152-   [0]=>
153-   string(0) ""
154- }
155- Error: 4096 - Object of class stdClass could not be converted to string, %s(73)
156- 
157- Arg value  
158- Error: 8192 - Function spliti() is deprecated, %s(74)
159- Error: 2 - spliti() expects parameter 2 to be string, object given, %s(74)
160- NULL
161- 
162- Arg value  
163- Error: 8192 - Function spliti() is deprecated, %s(74)
164- array(1) {
165-   [0]=>
166-   string(0) ""
167- }
168- 
169- Arg value  
170- Error: 8192 - Function spliti() is deprecated, %s(74)
171- array(1) {
172-   [0]=>
173-   string(0) ""
174- }
175- Done