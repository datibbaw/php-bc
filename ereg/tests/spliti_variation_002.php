<?php
/* Prototype  : proto array spliti(string pattern, string string [, int limit])
 * Description: spliti string into array by regular expression 
 * Source code: ext/standard/reg.c
 * Alias to functions: 
 */

function test_error_handler($err_no, $err_msg, $filename, $linenum, $vars) {
	echo "Error: $err_no - $err_msg, $filename($linenum)\n";
}
set_error_handler('test_error_handler');

echo "*** Testing spliti() : usage variations ***\n";

// Initialise function arguments not being substituted (if any)
$pattern = 'r|j|E';
$limit = 5;

//get an unset variable
$unset_var = 10;
unset ($unset_var);

//array of values to iterate over
$values = array(

      // int data
      0,
      1,
      12345,
      -2345,

      // float data
      10.5,
      -10.5,
      10.1234567e10,
      10.7654321E-10,
      .5,

      // array data
      array(),
      array(0),
      array(1),
      array(1, 2),
      array('color' => 'red', 'item' => 'pen'),

      // null data
      NULL,
      null,

      // boolean data
      true,
      false,
      TRUE,
      FALSE,

      // empty data
      "",
      '',

      // object data
      new stdclass(),

      // undefined data
      $undefined_var,

      // unset data
      $unset_var,
);

// loop through each element of the array for string

foreach($values as $value) {
      echo "\nArg value $value \n";
      var_dump( spliti($pattern, $value, $limit) );
};

echo "Done";
?>
