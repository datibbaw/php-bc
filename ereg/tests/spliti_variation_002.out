*** Testing spliti() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(64)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(67)

Arg value 0 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(1) "0"
}

Arg value 1 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(1) "1"
}

Arg value 12345 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(5) "12345"
}

Arg value -2345 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(5) "-2345"
}

Arg value 10.5 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(4) "10.5"
}

Arg value -10.5 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(5) "-10.5"
}

Arg value 101234567000 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(12) "101234567000"
}

Arg value 1.07654321E-9 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(2) {
  [0]=>
  string(10) "1.07654321"
  [1]=>
  string(2) "-9"
}

Arg value 0.5 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(3) "0.5"
}
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)

Arg value Array 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)

Arg value Array 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)

Arg value Array 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)

Arg value Array 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)

Arg value Array 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
Error: 2 - preg_split() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
bool(false)

Arg value  
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(0) ""
}

Arg value  
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(0) ""
}

Arg value 1 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(1) "1"
}

Arg value  
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(0) ""
}

Arg value 1 
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(1) "1"
}

Arg value  
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(0) ""
}

Arg value  
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(0) ""
}

Arg value  
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(0) ""
}
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/spliti_variation_002.php(73)

Arg value  
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
Error: 2 - preg_split() expects parameter 2 to be string, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(198)
bool(false)

Arg value  
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(0) ""
}

Arg value  
Error: 16384 - Function spliti() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(192)
array(1) {
  [0]=>
  string(0) ""
}
Done